src/main/java/hydroguard/api/mock/controllers/reoccurringtask/ReoccurringTaskController.java

```diff
@@ -32,7 +32,6 @@ public void init() {
     public ReoccurringTasksDTO getAllReoccurringTasks(
             @RequestParam(defaultValue = "1") int page,
             @RequestParam(defaultValue = "10") int pageSize,
-            @RequestParam(required = false) UUID userId,
```

Comment:
waarom is heir userId gedelete. en dan niet in taskcontroller

Reply:
eeeeh, omdat ik chaotisch bezig ben ig. zal ook meteen ff aanpassen